src/domains/tasks/CreateTask/types.tsx

```diff
@@ -4,6 +4,7 @@ import { TaskSchema } from "@Tasks/common/types";
 export const CreateTaskFormSchema = TaskSchema.omit({
   id: true,
   customerId: true,
+  paymentStatus: true
 });
```

Comment:
As discussed, this works like the TypeScript `Omit<T, Keys>` utility type.

We take the TaskSchema, and tell zod to remove the `id`, `customerId` and `paymentStatus` fields.

This allow us to reuse the rest of the zod schema for the `CreateTaskPage` input validation.